src/script/view_model/ImageDetailViewViewModel.ts

```diff
@@ -178,6 +180,12 @@ export class ImageDetailViewViewModel {
     });
   };
 
+  readonly hadndlePressOnClose = (event: KeyboardEvent): void => {
```

Comment:
these two methods are also the same.. i guess they can stay 🤷‍♂️

Reply:
typo on the naming here though! 
```suggestion
  readonly handlePressOnClose = (event: KeyboardEvent): void => {
```